tools/system_calls.py

```diff
@@ -139,3 +163,38 @@ def get_screen():
     buffered_screen = BytesIO()
     screen.save(buffered_screen, format='JPEG')
     return base64.b64encode(buffered_screen.getvalue()).decode('utf-8')
+
+
+def url_history(url):
+    """Stores 10 last casted url
```

Comment:
Maybe 10 should not be hardcoded. 
Would it be possible to store history length in common.py and utilize this bar?